packages/db/src/translate/collections.ts

```diff
@@ -1,6 +1,37 @@
-import { NewCollectionRecord } from '@cumulus/types/api/collections';
-import { PostgresCollection } from '../types/collection';
+import { NewCollectionRecord, CollectionRecord } from '@cumulus/types/api/collections';
+import { PostgresCollection, PostgresCollectionRecord } from '../types/collection';
+const { removeNilProperties } = require('@cumulus/common/util');
```

Comment:
Nit: I kinda dislike using this from `common` because it props up a dependency on that package when really `removeNilProperties` is just doing `omitBy(record, isNil)`, so it doesn't seem much worse to just use those functions from lodash directly

Reply:
I think I agree, but I think in the interests of consistency if it's truly a nit I'm going to pun that to a future PR to wholesale make that change.